c2cgeoportal/scaffolds/create/project.yaml.mako_tmpl

```diff
@@ -1,9 +1,8 @@
 project_folder: {{project}}
 project_package: ${package}
 host: ${host}
-checker_path: /${instanceid}/wsgi/check_collector?
-checker_protocol: http
-checker_disable_ssl_certificate_validation: False
+# Configuration option for checker (starting GMF 2.2.5)
+checker_url: https://${host}/${instanceid}/wsgi/check_collector?
```

Comment:
Why changing this file?

Reply:
Instead of checker_path, checker_url should now be used in project.yaml (but checker_path will still work like before when used in project configuration)

Reply:
But it add something to migrate with possible side effect :-/